langchain/chat_models/base.py

```diff
@@ -183,9 +183,17 @@ def __call__(
             raise ValueError("Unexpected generation type")
 
     def call_as_llm(self, message: str, stop: Optional[List[str]] = None) -> str:
-        result = self([HumanMessage(content=message)], stop=stop)
+        return self.predict(message, stop=stop)
+
+    def predict(self, text: str, stop: Optional[List[str]] = None) -> str:
```

Comment:
There are multiple completion modes for LLMs. Should we accept an option parameter here?

Reply:
what do you mean by multiple completion modes?